app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -1,7 +1,75 @@
 (function (GOVUK) {
   GOVUK.GDM.analytics.pageViews = {
     'init': function () {
+      this.setCustomDimensions();
       GOVUK.analytics.trackPageview();
+    },
+
+    'setCustomDimensions': function() {
+      var search = GOVUK.GDM.analytics.location.search(),
+          filterGroupDimension = [],
+          numberOfOpportunites,
+          dimensions,
+          pairs,
+          pair,
+          i,
+          j;
+
+      // check that we're on the catalogue page for opportunites
+      if (GOVUK.GDM.analytics.location.pathname() === "/digital-outcomes-and-specialists/opportunities") {
```

Comment:
I know this is a broader thing and partly built into the _way we do things_, but hardcoding the exact url path here isn't tremendously DRY (don't repeat yourself) - changing an url route will need to be reflected here to be picked up properly (and will lead to a silent failure if someone forgets to do it).